web-server/sgx-wallet-impl/src/wallet_operations/update_otp_number.rs

```diff
@@ -23,6 +23,9 @@ pub fn update_otp_phone_number(request: &UpdateOtpPhoneNumber) -> UpdateOtpPhone
                 ))
             }
             UnlockWalletError::IoError(err) => UpdateOtpPhoneNumberResult::Failed(err.to_string()),
+            UnlockWalletError::AccountLocked => {
```

Comment:
Why is there error handling for account lock in `update_opt_number` when a user would not be able to login if the person has tried logging in more than 3 times and has not reset their pin?

Reply:
the function `update_otp_phone_number` in update_otp_number.rs uses the method `unlock_wallet` and then `mutates` the otp number. Upon unlocking wallet, the user may experience Account Lock.